scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -23,19 +23,91 @@ source "$(dirname "${0}")"/../functions
 L_PWD=$(pwd)
 cd "${DEVON_IDE_HOME}" || exit 255
 
+# Install Eclipse plugins by feature group id
 # $1: plugin feature group ID
 # $2: plugin URL (update site)
 function doInstallEclipsePlugin() {
-  local plugin="${1/\.feature\.group*/}"
-  echo "Installing eclipse plugin ${plugin} from ${2}"
-  # shellcheck disable=SC2086
-  "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -repository "${2}" -installIU "${1}" -vmargs ${ECLIPSE_VMARGS}
-  result=${?}
-  if [ "${result}" != 0 ]
+  # If plugin_name argument ${3} is passed
+  if [ -n "${3}" ]
   then
-    doEchoAttention "Installation of plugin ${plugin} failed. Please try to install manually or rerun setup (devon eclipse setup)."
+    echo "Installing Eclipse plugin '${3}' from URL ${2}"
+  else
+    echo "Installing Eclipse plugin '${1}' from URL ${2}"
   fi
-  return "${result}"
+
+  if [ -z "${1}" ]
+  then
+    doEcho "No plugins to install."
+    return
+  fi
+
+  local plugin_dependencies
+  plugin_dependencies=$(doCheckPluginsInstalled "${1}")
+
+  # Install plugins and write debug info into temp file
+  local debug_info_file
+  debug_info_file=$(mktemp)
```

Comment:
So far eclipse plugin stuff gets logged to `software/eclipse/configuration/*.log`.
Why don't we keep such location and also preserve the log instead of deleting it (see `rm` command below)?

Reply:
I've removed the `rm` command, because the file is going to be removed either way even without users intervention.